lib/gui/etcher.js

```diff
@@ -51,17 +51,7 @@ electron.app.on('ready', () => {
   });
 
   // Prevent flash of white when starting the application
```

Comment:
Yep, the method is different (and doesn't involve that hacky timeout we had before), but the intention is the same.